sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/implementation/RoleAssignmentHelper.java

```diff
@@ -94,7 +94,7 @@ public RoleAssignmentHelper withAccessTo(final String scope, final BuiltInRole a
                     .withBuiltInRole(asRole)
                     .withScope(resourceScope)
                     .createAsync()
-                    .last()
+                    .cast(Indexable.class)
```

Comment:
Why need a `cast`? Does it need `Flux<Indexable>`?

Reply:
Some are `dependency` or `post dependency`, some are in test for `Flux`